api/types.go

```diff
@@ -293,6 +293,36 @@ type EmbeddingResponse struct {
 	Embedding []float64 `json:"embedding"`
 }
 
+// TokenizeRequest is the request sent by [Client.Tokenize].
+type TokenizeRequest struct {
+	Model string `json:"model"`
+	Text  string `json:"text"`
+
+	// KeepAlive controls how long the model will stay loaded in memory following
+	// this request.
+	KeepAlive *Duration `json:"keep_alive,omitempty"`
```

Comment:
This shouldn't be required in the API – we should load the model in vocab only mode.